tests/utils.go

```diff
@@ -5316,3 +5316,13 @@ func GetIdOfLauncher(vmi *v1.VirtualMachineInstance) string {
 
 	return strings.TrimSpace(podOutput)
 }
+
+func ExecuteCommandOnNodeThroughVirtHandler(virtCli kubecli.KubevirtClient, node k8sv1.Node, command []string) (stdout string, stderr string, err error) {
```

Comment:
I'm surprised there isn't already a function that does that!
At first glance I can't find one, however `RemoveHostDiskImage()` seems to do the same thing but with a cool trick to get the virt-handler for the node, probably worth having a look!

Reply:
Thanks for the tip.
Done :)